.releaserc.js

```diff
@@ -45,11 +45,6 @@ module.exports = {
       "@google/semantic-release-replace-plugin",
       {
         "replacements": [
-          {
-            "files": ["packages/*/package.json"],
-            "from": "\"mongodb-memory-server\": \".*\"",
-            "to": "\"mongodb-memory-server\": \"${nextRelease.version}\"",
-          },
```

Comment:
because, from my knowledge this was to replace `dependencies: { mongodb-memory-server: "x.x.x" }` with the new version, and because that package is not depended anymore (now all depend on `-core` to fix the mentioned issues) so i removed that replace (unnecessary)

Reply:
Ah, ok.

So if all `global` packages now depends from `mongodb-memory-server-core` we need to replace that code on:
```diff
             "files": ["packages/*/package.json"],
-            "from": "\"mongodb-memory-server\": \".*\"",
+            "from": "\"mongodb-memory-server-core\": \".*\"",
-            "to": "\"mongodb-memory-server\": \"${nextRelease.version}\"",
+            "to": "\"mongodb-memory-server-core\": \"${nextRelease.version}\"",
```
for keeping in sync all global packages with new published version of core package

Reply:
Otherwise, all global packages would be frozen on incorrect `mongodb-memory-server-core` in their package.json files

Reply:
no, because that already existed before, so i only removed the one replacing `mongodb-memory-server:` (because that dosnt apply anymore)

look here for an reference: https://github.com/nodkz/mongodb-memory-server/blob/beta/.releaserc.js#L43-L60

Reply:
Sorry, I missed that!